modules/linux-web-app/r-appservice.tf

```
@@ -122,6 +122,134 @@ resource "azurerm_linux_web_app" "app_service_linux" {
     }
   }
 
+  dynamic "auth_settings_v2" {
+    for_each = lookup(var.auth_settings_v2, "auth_enabled", false) ? [var.auth_settings_v2] : []
+    content {
+      auth_enabled                            = lookup(auth_settings_v2.value, "auth_enabled", false)
+      runtime_version                         = lookup(auth_settings_v2.value, "runtime_version", "~1")
+      config_file_path                        = lookup(auth_settings_v2.value, "config_file_path", null)
+      require_authentication                  = lookup(auth_settings_v2.value, "require_authentication", null)
+      unauthenticated_action                  = lookup(auth_settings_v2.value, "unauthenticated_action", "RedirectToLoginPage")
+      default_provider                        = lookup(auth_settings_v2.value, "default_provider", "azureactivedirectory")
+      excluded_paths                          = lookup(auth_settings_v2.value, "excluded_paths", null)
+      require_https                           = lookup(auth_settings_v2.value, "require_https", true)
+      http_route_api_prefix                   = lookup(auth_settings_v2.value, "http_route_api_prefix", "/.auth")
+      forward_proxy_convention                = lookup(auth_settings_v2.value, "forward_proxy_convention", "ForwardProxyConventionNoProxy")
```

`expected auth_settings_v2.0.forward_proxy_convention to be one of [NoProxy Custom Standard], got ForwardProxyConventionNoProxy`

```suggestion
      forward_proxy_convention                = lookup(auth_settings_v2.value, "forward_proxy_convention", "NoProxy")
```

To avoid the following error

```
expected auth_settings_v2.0.forward_proxy_convention to be one of [NoProxy Custom Standard], got ForwardProxyConventionNoProxy
```

---

Tks For this. It seems Azure Provider Documentation issue, because in documentation Says the Default it's ForwardProxyConventionNoProxy.